packages/mui-codemod/src/v5.0.0/base-use-named-exports.test/actual.js

```diff
@@ -0,0 +1,10 @@
+import Button from '@mui/base/Button';
+import Badge, { badgeClasses } from '@mui/base/Badge';
+import BaseSwitch from '@mui/base/Switch';
+import BaseSelect, { selectClasses } from '@mui/base/Select';
+import { Menu, MenuItem } from '@mui/base';
+import ClassNameConfigurator from '@mui/base/utils/ClassNameConfigurator';
```

Comment:
I would say this is not a valid import. We are leaking internal API. By convention we don't allow these imports and consider them private API.

Reply:
Good point. Excluding this will simplify the codemod code.